demos/browser/server.js

```diff
@@ -83,6 +83,32 @@ const server = require(protocol).createServer(options, async (request, response)
       response.write(JSON.stringify(attendeeInfo), 'utf8');
       response.end();
       log(JSON.stringify(attendeeInfo, null, 2));
+    } else if (request.method === 'POST' && request.url.startsWith('/createMeeting?')) {
```

Comment:
We haven't strictly been following REST semantics in the endpoints defined in this server, but `POST /createMeeting` could just be `POST /meeting`.

Reply:
Will update it